askama_rocket/src/lib.rs

```diff
@@ -6,11 +6,11 @@ pub use rocket::request::Request;
 use rocket::response::Response;
 pub use rocket::response::{Responder, Result};
 
-pub fn respond<T: Template>(t: &T, ext: &str) -> Result<'static> {
+pub fn respond<'r, 'o, T: Template>(t: &'r T, ext: &'r str) -> Result<'o> {
```

Comment:
Sorry, I'm still confused: why do we need `'o` here? Since there is no input parameter that uses `'o`, couldn't we keep it as is, with the return type using `'static` directly?

Reply:
We don't. Thank you for pointing that out. The lifetimes are part of Rocket's Responder signature. In our case we should be able to get away with elided param lifetimes and a static out lifetime.